src/components/form/Wrapper.styled.js

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+	witdh: 100vw;
+	height: 100vh;
+	display: flex;
+	justify-content: center;
+
+
+
+}
+button {
+background: rgba(21, 146, 230, 0.5);
+width: 170px;
+height: 40px;
+border: 1px solid rgba(255, 0, 0, 0.8);
+margin: 0.3rem;
+
+border-radius: 2px;
+font-size: 0.9rem;
+font-family: Arial, Helvetica, sans-serif;
+letter-spacing: 0.2rem;
+color: #000000;
+cursor: pointer;
+color:white;
+
+
+}
```

Comment:
Looks like your Prettier didn't work here, maybe because there are some more typos? Please check 😊

Reply:
is changed